roadmaps/2021.md

```diff
@@ -23,9 +23,38 @@ Nothing is definitive: the roadmap can be adapted depending on the need of the u
 
 ## Q2
 
-- [ ] Start some benchmarks
+- [X] Start some benchmarks
 - [X] Version the core-team repositories
-- [ ] Publish the core-team repositories to crates.io
-- [ ] Release the first RC of MeiliSearch v0.21.0 that integrates the new search engine (Transplant + Milli)
-- [ ] Release v0.21.0 of MeiliSearch
-
+- _Reported, waiting for internal decision_ Publish the core-team repositories to crates.io
+- [X] Release the first RC of MeiliSearch v0.21.0 that integrates the new search engine (Transplant + Milli)
+- _Reported to Q3_ Release v0.21.0 of MeiliSearch
+
+## Q3
+
+### Internal
+
+- [ ] Release v0.21.0
+- [ ] Publish core-team repositories (depending on internal decisions)
+  - [ ] tokenizer
+  - [ ] milli
+  - [ ] lmdb-rs
+- [ ] Cron job CI to run some SDKs tests
+- [ ] MeiliSearch (e.g. Transplant) documentation
+- [ ] Indexation benchmarks
+
+### New features
+
+- [ ] SortBy
+- [ ] Geo search
+- [ ] NDJSON
+- [ ] CSV
+- [ ] Improve analytics
+- [ ] Clean API
+  - [ ] Search
+  - [ ] Documents
+  - [ ] Updates
+  - [ ] Indexes
+
+### Optimization
+
+- [ ] Indexer
```

Comment:
```suggestion
```

Reply:
I moved the optimization line under the objective of improving indexing performance! Thanks, your work @curquiza!